woo-ecurring.php

```diff
@@ -113,7 +114,7 @@ function ecurring_wc_check_woocommerce_status() {
 
 add_action('plugins_loaded', function(){
 	$environmentChecker = new EnvironmentChecker();
-    if (!$environmentChecker->isMollieActive() || !$environmentChecker->isMollieMinimalVersion()) {
+    if (!$environmentChecker->isMollieActive()) { // || !$environmentChecker->isMollieMinimalVersion()
```

Comment:
Yeah, we need to find some solution to use Mollie plugin with not updated version while it's in development.